pkg/interface/chat/src/js/api.js

```diff
@@ -34,6 +36,17 @@ class UrbitApi {
   hall(data) {
     this.action("hall", "hall-action", data);
   }
+  
+  addPendingMessage(data) {
+    let appendDataObject = data;
+    appendDataObject.pending = true;
+    data = appendDataObject;
```

Comment:
given that you aren't changing the function scope, I believe you can just set `data.pending = true;`

Reply:
Removing the boolean setting at this level and doing it as we're preparing message as props in chat.js.